pkg/kubelet/kuberuntime/kuberuntime_sandbox_others.go

```diff
@@ -0,0 +1,49 @@
+//go:build !windows
+// +build !windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kuberuntime
+
+import (
+	v1 "k8s.io/api/core/v1"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func addNonWindowsRelatedContext(lc *runtimeapi.LinuxPodSandboxConfig, sc *v1.PodSecurityContext) {
```

Comment:
I don't _love_ the naming here, but I also can't think of anything meaningfully better off the top of my head.

Reply:
Since this is setting ``SecurityContext`` options in a ``LinuxPodSandboxConfig``, I've renamed this to ``addLinuxSecurityContext``. WDYT?